lib/utils/docker.coffee

```diff
@@ -27,7 +27,6 @@ exports.appendConnectionOptions = appendConnectionOptions = (opts) ->
 			signature: 'dockerPort'
 			parameter: 'dockerPort'
 			description: 'Docker daemon TCP port number (hint: 2375 for balena devices)'
-			alias: 'p'
```

Comment:
I've added comment https://github.com/balena-io/balena-cli/issues/1159#issuecomment-487644699 to consider preserving `-p` for `dockerPort`.